GliaWidgets/Sources/ViewController/Chat/ChatViewController.swift

```diff
@@ -227,8 +227,9 @@ class ChatViewController: EngagementViewController, MediaUpgradePresenter,
         switch type {
         case .chat:
             chatView.props = .init(header: props.chat)
-        case .secureTranscript:
+        case let .secureTranscript(allowsInput):
             chatView.props = .init(header: props.secureTranscript)
+            chatView.messageEntryView.isHidden = !allowsInput
```

Comment:
What happens with the messages when the message entry view is hidden? Do texts get hidden behind the unavailable alert?

Reply:
Yes, like so:
<img src="https://user-images.githubusercontent.com/3148347/225895076-ca1932c5-e119-40c8-b00b-ba1c498d29a3.png" width="428" height="926">